README.MD

```diff
@@ -246,28 +114,30 @@ spec:
     storageOSOperatorNamespace: storageos-operator-old
 ```
 
-For an example config file, see:
+For an example config file, see `config/samples/_v1_kubectlstorageosconfig.yaml`.
 
-`config/samples/_v1_kubectlstorageosconfig.yaml`
+## Enable TLS
 
-## TLS Enablement
+### Install ETCD and StorageOS with TLS enabled
 
-### Install ETCD and StorageOS With TLS Enabled
-
-`kubectl storageos install --include-etcd --etcd-tls-enabled`
+```bash
+kubectl-storageos install --include-etcd --etcd-tls-enabled
+```
 
-### Install StorageOS and Connect to an Existing TLS Enabled ETCD Cluster
+### Install StorageOS and connect to an existing TLS enabled ETCD cluster
 
-`kubectl storageos install --etcd-tls-enabled`
+```bash
+kubectl-storageos install --etcd-tls-enabled
+```
 
 In order for StorageOS to connect to ETCD over HTTPS, a k8s secret must exist in the `storageos-cluster-namespace` with valid ETCD credentials.
 This secret can be created like so:
 
 ```lang-none
-kubectl create secret generic <etcd-secret-name> -n <storageos-cluster-namespace> \
-                --from-file=etcd-client-ca.crt=path/to/ca.crt \
-                --from-file=etcd-client.crt=path/to/tls.crt \
-                --from-file=etcd-client.key=path/to/tls.key
+kubectl-create secret generic <etcd-secret-name> -n <storageos-cluster-namespace> \
```

Comment:
`kubectl-create`? I have never used that command.

Reply:
Me neither :D